src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -29,11 +29,21 @@ public WinRmToolResponse executeScript(List<String> commands) {
         return responseToWinRmToolResponse(response);
     }
 
+    public WinRmToolResponse executeCommand(String command) {
+        Response response = session.run_cmd(command);
+        return responseToWinRmToolResponse(response);
+    }
+
     public WinRmToolResponse executePs(List<String> commands) {
         Response response = session.run_ps(compileScript(commands));
         return responseToWinRmToolResponse(response);
     }
 
+    public WinRmToolResponse executePsCommand(String command) {
```

Comment:
Should we just overload the `executePs` method (with this one taking a string), rather than using a different name? 

Same for `executeCommand`: I'd prefer consistency of naming. If we think that `executeScript` was not the best name, then we can look at changing that name.